internal/pkg/service/stream/storage/statistics/repository/stats_schema.go

```diff
@@ -17,7 +17,9 @@ const (
 	// rollupSumKey contains the sum of all statistics from the object children that were deleted.
 	rollupSumKey = "_sum"
 	// rollupResetKey contains the sum of all statistics from the object children that are ignored.
-	rollupResetKey = "_reset"
+	// ~ char is important, in alphabetical order, the reset will always be at the end - after the statistics to which it belongs.
+	// We are using uint64 types, so we have to prevent underflow - the reset must be subtracted at the end.
+	rollupResetKey = "~reset"
 )
```

Comment:
This change will simplify the code.

Reply:
See ASCII table: https://www.asciitable.com/

Reply:
https://www.editpad.org/tool/alphabetizer

![image](https://github.com/user-attachments/assets/0586b277-07fa-4503-b463-da3a3b2e239c)